core/src/main/java/org/frankframework/processors/MonitoringPipeProcessor.java

```diff
@@ -76,13 +67,13 @@ protected PipeRunResult processPipe(PipeLine pipeLine, IPipe pipe, Message messa
 		try {
 			pipeRunResult = chain.apply(message);
 		} catch (PipeRunException pre) {
-			if (pe!=null) {
-				pe.throwEvent(IExtendedPipe.PIPE_EXCEPTION_MONITORING_EVENT);
+			if (pipe!=null) {
```

Comment:
Zou `pipe` ooit `null` kunnen zijn in `processPipe()`?

Ik zie dat er al null-checks op `pipe` inzitten maar dat is absoluut onlogisch.

Ook `pipeLineSession` zou nooit `null` mogen kunnen zijn in deze code, daar zie ik ook `null` checks op.
Andere PipeProcessor implementaties hebben die null-checks op pipe en pipeLineSession ook niet en ze zijn allemaal genest -- haal die null-checks maar weg hier!